Bottom: 59fc607250a481aac5da87d4296f1783f949792b
Top:    642718307250c62d80e83e8ad81aafb696296a92
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-31 02:14:19 -0300

Refresh of manual-document-mt-safety-stdio.patch

---

diff --git a/manual/stdio.texi b/manual/stdio.texi
index dad1072..9ffe87b 100644
--- a/manual/stdio.texi
+++ b/manual/stdio.texi
@@ -2680,7 +2680,7 @@ pointer @var{ap}.
 @c outchar ok
 @c outstring ok
 @c PAD ok
-@c __printf_fp glocale memleak
+@c __printf_fp glocale, asmalloc, memleak
 @c __printf_fphex glocale
 @c __readonly_area
 @c   [GNU/Linux] fopen, strtoul, free
